app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
> /home/travis/build/EventFahrplan/EventFahrplan/app/src/main/res/values-ja/strings.xml:48:127: Error: The prefix "xliff" for element "xliff:g" is not bound.

You are missing the namespace declaration here. Please change it to:

``` xml
<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
```

Reply:
oh, indeed.
I added it in the next commit: "add namespace declaration"